src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -52,11 +56,11 @@ var ChartsPanel = React.createClass({
 
     getInitialState() {
         return {
-            chartType : SCATTER,
-            optionsShown : true,
+            chartType: SCATTER,
+            optionsShown: true,
             widthPx: 700,
             heightPx: 300,
-            debouncedResize: debounce(this.onResize, 500, {'leading':false, 'trailing':true})};
+            debouncedResize: debounce(this.onResize, 200, {'leading':false, 'trailing':true, 'maxWait':10000})};
```

Comment:
I think you are right. I was trying to figure out why I get at least two resizes. (I wanted to have just one, because it is expensive.) One of the guesses was that the default maxWait is too short. I think it's canceling the previous debounce that fixed the problem.